docs/source/tutorials-and-examples/tutorial.rst

```diff
@@ -312,7 +312,7 @@ in MLflow saved the model as an artifact within the run.
 
       .. code-block:: bash
 
-          curl -X POST -H "Content-Type:application/json; format=pandas-split" --data '{"columns":["alcohol", "chlorides", "citric acid", "density", "fixed acidity", "free sulfur dioxide", "pH", "residual sugar", "sulphates", "total sulfur dioxide", "volatile acidity"],"data":[[12.8, 0.029, 0.48, 0.98, 6.2, 29, 3.33, 1.2, 0.39, 75, 0.66]]}' http://127.0.0.1:1234/invocations
+          curl -X POST -H "Content-Type:application/json; format=pandas-split" --data "{\"columns\":[\"alcohol\", \"chlorides\", \"citric acid\", \"density\", \"fixed acidity\", \"free sulfur dioxide\", \"pH\", \"residual sugar\", \"sulphates\", \"total sulfur dioxide\", \"volatile acidity\"],\"data\":[[12.8, 0.029, 0.48, 0.98, 6.2, 29, 3.33, 1.2, 0.39, 75, 0.66]]}" http://127.0.0.1:1234/invocations
```

Comment:
@extrospective Thanks for the contribution! Can you fix these lines as follows because showing only the escaped command would make it difficult to understand what the json data looks like?
 
```
# On Linux and macOS
curl -X ...

# On Windows
curl -X ...
```

Reply:
I thought about it, but decided since the purpose of the tutorial is not to introduce curl but to teach mlflow that it was not worth the added complexity.

Do you feel confident this is a better solution from the user's perspective?

If so I will implement as suggested.

Reply:
@dmatrix What do you think?

Reply:
@extrospective thanks for fixing this. From the developer's perspective (and a reader of both UNIX and Windows), having two
separate callouts make it readable. It's not an uncommon practice in developer documentation to have separate callouts of CLI syntax between different operating systems. So I would lean toward:

```
# On Linux and macOS
curl -X ...

# On Windows
curl -X ...
```

Reply:
I appreciate the thoughtful feedback.  

Since the change is compatible for Linux and macOS I had not come to the same conclusion.  It's just a less readable format with quoting.

But it might not be worth the time to opine further.  I will implement as proposed.  It will work for end users.